src/algebra/lie/of_associative.lean

```diff
@@ -37,28 +37,28 @@ section of_associative
 
 variables {A : Type v} [ring A]
 
-namespace ring_commutator
+namespace ring
 
 /-- The bracket operation for rings is the ring commutator, which captures the extent to which a
 ring is commutative. It is identically zero exactly when the ring is commutative. -/
 @[priority 100]
 instance : has_bracket A A := ⟨λ x y, x*y - y*x⟩
 
-lemma commutator (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
+lemma has_bracket_def (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
```

Comment:
Would `lie_def` be more in line with names like `lie_add` etc?
`lie_eq` or `lie_eq_commutator` also seem good to me.

Reply:
Yes, and I like `lie_def` best out of the above. Updated.